download/_posts/2015-05-05-2.12.0-M1.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Scala 2.12.0-M1
+start: 05 May 2015
+layout: downloadpage
+release_version: 2.12.0-M1
+release_date: "May 05, 2015"
+show_resources: "true"
+permalink: /download/2.12.0-M1.html
+requirements: "This Scala software distribution can be installed on any Unix-like or Windows system. It requires the Java runtime version 1.6 or later, which can be downloaded <a href='http://www.java.com/'>here</a>."
```

Comment:
Could you please add a note to the release checklist for M2 to remind us to edit this to require 1.8?